main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: robot
+//  action 1: the robot moves forward
+robot.forward();
+//  action 2: the robot turns left 90 degrees 
+robot.left_90();

```

Comment:
I'm marking this because:
- either your plain-english description is not an action, but is an ability or describes a state
- or the plain-english description is fine but the function name doesn't match.
- The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise